cocli/README.md

```diff
@@ -19,408 +19,68 @@ To get a list of the supported shells, do:
 ```
 $ cocli completion --help
 ```
+# CoRIM Handling
+A CoRIM consists of either one or more CoMID and/or CoSWID and/or CoTS. The below
+picture and instructions provides a step by step approach to handle CoRIMs.
+
+``` mermaid
```

Comment:
The picture in its present form does not achieve its intent (which is a good one).  Instead of adding a new picture, maybe refactor the one that already exists.

Reply:
The picture presented here depicts the high level block how a CoCli commands impact the individual CoRIM Blocks.

I would let Hannes see how he feels this. I do not want to re-create the same daigram, which already has a good snapshot!

Reply:
> The picture presented here depicts the high level block how a CoCli commands impact the individual CoRIM Blocks.
> 
> I would let Hannes see how he feels this. I do not want to re-create the same daigram, which already has a good snapshot!

The picture mixes the structural aspects of CoRIM with a depiction of the available manipulation tools.  The problem is that these two layers of information are conflated in the same visual space without clear typographical conventions that would help to disentangle the meaning - while keeping the relation apparent.

I suggest instead you split the other, pre-existing picture around each command/subcommand to show the different stages of the processing chain.

Reply:
Reusing the same picture risks duplication.

I am happy to make the existing picture more clear by color coding boxes for Commands different than that of the individual CoMID/CoSWID/CoTS. 

Also, I suggest we rely more on the external contributors/reviews like Hannes to provide the feedback, as we both risk
from bias's in our own directions, which may not necessarily be very helpful!

Reply:
When we put out a PR we rely on whatever feedback we can muster.  Besides, ignoring someone's feedback makes that someone less inclined to provide feedback next time...

Reply:
No intention to `ignore` anyone's feedback! 
I really meant, we seek opinion from the individual who originally found the problem and reported him! Does the individual find the PR changes meaningful and does it address the concern he had originally put in ?

Reply:
Addressed the concern by modifying the picture, to indicate a clear seperation of `CoRIM`  block and the `command` block!